test.py

```diff
@@ -0,0 +1,17 @@
+import random
+import pandas as pd
+import numpy as np
+
+
+def test(cols_size, rows_size):
+    table = []
+
+    for i in range(rows_size):
```

Comment:
Even if this function is really small/short, do not hesitate to add some comments for different parts. Comments are really helpful if most of the time. One think to remember: do not put a lot of text into comments as we should treat them as a little help during code understanding process. You can also mark a block of code for particular comments, for example:

```python
# Note: generate some special value and show the last entry
some_variable = [x for x in (some_iterator)]
print(some_variable[-1]
# --- end note
```

Ofc. it depends on particular programmer style, so do not bother mine ;) Just find yours!